From 046dfac63982ac6d973d34a7c8e8596780eb2b0c Mon Sep 17 00:00:00 2001
From: Naseer Ahmed <naseer@codeaurora.org>
Date: Tue, 6 May 2014 17:42:29 -0400
Subject: [PATCH 1/3] copybit: Export c2d2 headers from display HAL
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Some builds may not have this header exported from graphics.
Export it from the display HAL.

Change-Id: Ic7f3c4bd1d3ff16c7c44518d689be1c67bb15295
Signed-off-by: D. Andrei Măceș <andrei@unlegacy-android.org>
---
 libcopybit/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libcopybit/Android.mk b/libcopybit/Android.mk
index fcbc77500..8aaf0a353 100644
--- a/libcopybit/Android.mk
+++ b/libcopybit/Android.mk
@@ -17,7 +17,7 @@ include $(LOCAL_PATH)/../common.mk
 
 include $(CLEAR_VARS)
 LOCAL_COPY_HEADERS_TO         := $(common_header_export_path)
-LOCAL_COPY_HEADERS            := copybit.h copybit_priv.h
+LOCAL_COPY_HEADERS            := copybit.h copybit_priv.h c2d2.h
 include $(BUILD_COPY_HEADERS)
 
 include $(CLEAR_VARS)
-- 
2.17.0

